compiler/src/dotty/tools/dotc/interactive/Completion.scala

```diff
@@ -53,9 +53,9 @@ object Completion:
     val completionContext = Interactive.contextOfPath(tpdPath).withPhase(Phases.typerPhase)
     inContext(completionContext):
       val untpdPath = Interactive.resolveTypedOrUntypedPath(tpdPath, pos)
-      val mode = completionMode(untpdPath, pos, forSymbolSearch = true)
```

Comment:
You can now remove forSymbolSearch

Reply:
Ach, right! I knew I wanted to change that, but then I forgot.